content/members/apapadopoulou.md

```diff
@@ -1,5 +1,5 @@
 title: Angeliki - Papadopolou date: 20200528 joined_date: 20200529 category: members member_category: Researchers id: apapadopoulou
```

Comment:
Should be
```
title: Angeliki - Papadopolou 
date: 20200528 
joined_date: 20200529 
category: members 
member_category: Researchers 
id: apapadopoulou
```
rather than
```
title: Angeliki - Papadopolou date: 20200528 joined_date: 20200529 category: members member_category: Researchers id: apapadopoulou
```